fv3_cap.F90

```diff
@@ -1011,7 +993,7 @@ subroutine InitializeAdvertise(gcomp, rc)
       enddo
     endif
 
-    ! --- advertise Fields in importState and exportState -------------------
```

Comment:
I should have been more clear. I just meant the ascii art at the top of the subroutine. In general we don't want to mess with the comments in the code.

However, let's just go with this one.